src/renderer/src/helpers/UtilsHelper.ts

```diff
@@ -116,8 +116,8 @@ export class UtilsHelper {
     if (hash.length === 0) throw new Error('Invalid hash')
 
     const mandatoryColors: Record<string, string> = {
-      ef4073a0f2b305a38ec4050e4d3d28bc40ea63f5: '#56f33f',
-      d2a4cff31913016155e38e474a2c06d08be276cf: '#02c797',
+      '0xef4073a0f2b305a38ec4050e4d3d28bc40ea63f5': '#56f33f',
+      '0xd2a4cff31913016155e38e474a2c06d08be276cf': '#02c797',
```

Comment:
A better solution for this would be to leave the hash without the 0x and create a new method within UtilsHelper called normalizeHash that remove the 0x from the string and call it before checking the object


```suggestion
       static normalizeHash(hash: string) {
        return hash.replace('0x', '').toLowerCase()
      }
      
     static generateTokenColor(hash: string) {
      const normalizedHash = this.normalizeHash(hash)
      
      const mandatoryColors: Record<string, string> = {
      '0xef4073a0f2b305a38ec4050e4d3d28bc40ea63f5': '#56f33f',
      '0xd2a4cff31913016155e38e474a2c06d08be276cf': '#02c797',
      }
      
      if (mandatoryColors[normalizedHash]) return mandatoryColors[normalizedHash]
      ... rest of method
```